psi4/src/psi4/libplugin/load_plugin.cc

```diff
@@ -60,9 +59,9 @@ plugin_info plugin_load(std::string& plugin_pathname)
         throw PSIEXCEPTION(msg);
     }
 
-//    boost::filesystem::path pluginPath(plugin_pathname);
-//    boost::filesystem::path pluginStem = pluginPath.stem();
-//    info.name = pluginStem.string();
+    //    boost::filesystem::path pluginPath(plugin_pathname);
```

Comment:
Can these three commented lines be removed?

Reply:
good catch.  done!